src/render_graph/screen_info_node.rs

```diff
@@ -30,18 +24,17 @@ impl Node for ScreenInfoNode {
     fn update(
         &mut self,
         _world: &World,
-        resources: &Resources,
         render_context: &mut dyn RenderContext,
         _input: &ResourceSlots,
         _output: &mut ResourceSlots,
     ) {
         const BUFFER_SIZE: usize = std::mem::size_of::<[f32; 4]>();
 
         // Fetch resources
-        let window_created_events = resources.get::<Events<WindowCreated>>().unwrap();
-        let window_resized_events = resources.get::<Events<WindowResized>>().unwrap();
-        let windows = resources.get::<Windows>().unwrap();
-        let mut render_resource_bindings = resources.get_mut::<RenderResourceBindings>().unwrap();
+        let window_created_events = _world.get_resource::<Events<WindowCreated>>().unwrap();
+        let window_resized_events = _world.get_resource::<Events<WindowResized>>().unwrap();
+        let windows = _world.get_resource::<Windows>().unwrap();
+        let mut render_resource_bindings = _world.get_resource_mut::<RenderResourceBindings>().unwrap();
```

Comment:
World is not burrowed as mut anymore so this is invalid. I have no idea if we can get the RenderResourceBindings any other way...